FinSearchUnified/Helper/StaticHelper.php

```diff
@@ -193,6 +196,17 @@ public static function isCategoryPage(Enlight_Controller_Request_Request $reques
             array_key_exists('sCategory', $request->getParams());
     }
 
+    /**
+     * @param Enlight_Controller_Request_Request $request
+     *
+     * @return bool
+     */
+    public static function isManufacturerPage(Enlight_Controller_Request_Request $request)
+    {
+        return $request->getControllerName() === 'listing' &&
+            ($request->getActionName() === 'manufacturer' || $request->getActionName() === 'listingCount');
```

Comment:
This is not the exact same as the one in `Frontend`. Why was the `$request->getActionName() === 'listingCount'` part added?

Reply:
Sorry for that. Because when I worked on that task, on Manufacturer page, initial request was handled by 'manufacturer' action, but when we change the sort, that request was handled by 'listingCount' action.
I try today to reproduce this case but now every request handled by 'manufacturer' action. I will remove this additional condition in next commit.